### ISSUE TITLE
Everything is enlarged when I added the <wysiwyg> on the template

### ISSUE THREAD

**Issue by alvindimla** [chmln/vue-wysiwyg#77]
Everything is enlarged and I can only see the toolbar icons which takes up the whole page.

**Comment by amurrell** [comment]
You need to ensure that the wysiwyg css import is loading - it sounds like it's not.

**Comment by MrSunshyne** [comment]
@alvindimla  Yeah, i encountered the same issue. Importing the css in main.js fixed this for me : 

> main.js
``` 
import "vue-wysiwyg/dist/vueWysiwyg.css";
```
